### PR TITLE
Invisible Chasms

### PR DESCRIPTION
**Pull request by Aidenkrz**
does what it says on the tin, using this for rooftops.

### CODE REVIEW

File: Resources/Prototypes/_SSS/markers.yml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  id: FloorChasmEntityInvisible
+  parent: MarkerBase
+  placement:
+    mode: SnapgridCenter
+    snap:
+    - Wall
+  components:
+  - type: Sprite
+    layers:
+      - sprite: _SSS/markers.rsi # {
```
Comment (Simyon264) [inline Resources/Prototypes/_SSS/markers.yml:25]: Empty comment?

File: Resources/Prototypes/_SSS/markers.yml [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        state: chasm
+        shader: unshaded
+  - type: AtmosFixMarker
+  - type: Chasm
+  - type: StepTrigger
+    requiredTriggeredSpeed: 0
+    intersectRatio: 0.4
+    blacklist:
+      tags:
+      - Catwalk
```
Comment (Simyon264) [inline Resources/Prototypes/_SSS/markers.yml:35]: AFAIK that means you can build over them with catwalks, which if they are invisble and only supposed to be mapped for death pits you should probably not be able to build over them.

File: Resources/Prototypes/_SSS/markers.yml [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  id: FloorChasmEntityInvisibleCorner
+  parent: MarkerBase
+  placement:
+    mode: SnapgridCenter
+    snap:
+    - Wall
+  components:
+  - type: Sprite
+    layers:
+      - sprite: _SSS/markers.rsi # {
```
Comment (Simyon264) [inline Resources/Prototypes/_SSS/markers.yml:69]: empty comment again

File: Resources/Prototypes/_SSS/markers.yml [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+  id: FloorChasmEntityInvisibleSide
+  parent: MarkerBase
+  placement:
+    mode: SnapgridCenter
+    snap:
+    - Wall
+  components:
+  - type: Sprite
+    layers:
+      - sprite: _SSS/markers.rsi # {
```
Comment (Simyon264) [inline Resources/Prototypes/_SSS/markers.yml:113]: smh again

**Review by Simyon264 — CHANGES_REQUESTED** [review]

**Comment by Aidenkrz** [comment]
yeah i should also just be parenting off the normal one for the side and corner ones, I was tripping

**Review by Simyon264 — APPROVED** [review]

**Pull request merged**